source/fundamentals/connection/mongoclientsettings.txt

```diff
@@ -259,26 +259,22 @@ settings to modify the driver's behavior:
 
    * - ``serverSelector()``
      - Adds a server selector to apply before server selection.
-
+  
    * - ``srvHost()``
-     - Sets the host name to use to look up an SRV DNS record to find the
-       MongoDB hosts.
-
-       .. note::
-
-          When setting ``srvHost``, the driver does not process any
-          associated TXT records associated with the host.
-
-          If you want to enable the processing of TXT records, you must
-          specify the SRV host in the connection string using the
-          ``applyConnectionString()`` method.
-
-          .. code-block:: java
-             :emphasize-lines: 3
-
-             MongoClient mongoClient = MongoClients.create(
-                MongoClientSettings.builder()
-                   .applyConnectionString(new ConnectionString("mongodb+srv://host1.acme.com")))
+     - | Sets the host name to use to look up an SRV DNS record to find the
+         MongoDB hosts. When setting ``srvHost``, the driver does not process any associated 
```

Comment:
```suggestion
         MongoDB hosts. 
         
         When setting ``srvHost``, the driver does not process any associated 
```
Suggestion: Separate the method definition from the implementation details. Think it looks a bit crowded as is and seems to match the format of the next table entry.